pennylane_lightning_kokkos/lightning_kokkos.py

```diff
@@ -101,6 +105,72 @@ def syncD2H(self, use_async=False):
         self._kokkos_state.DeviceToHost(self._state.ravel(order="C"))
         self._pre_rotated_state = self._state
 
+    """
+    @staticmethod
+    def kokkos_config_dict():
+        info_str = kokkos_config_info()["All_Info"]
+
+        query_keys = [
+            "Kokkos Version",
+            "Compiler",
+            "Arch",
+            "Atomics",
+            "Vectorization",
+            "Memory",
+            "Options",
+            "Backend",
+            "Runtime Config",
+        ]
+
+        config_meta_map = {}
+
+        for i in range(len(query_keys)):
+            key = query_keys[i]
+            config_meta_map[key] = {}
+
+        info_str_list = info_str.split("\n")
+
+        looped_keys = []
+
+        for i in range(len(info_str_list)):
+            tmp_str = info_str_list[i]
+            looped_len = len(looped_keys)
+            if looped_len == 0 and query_keys[looped_len] in tmp_str:
+                looped_keys.append(query_keys[looped_len])
+                config_meta_map[query_keys[looped_len]] = tmp_str
+
+            elif looped_len == 1 and query_keys[looped_len] in tmp_str:
+                looped_keys.append(query_keys[looped_len])
+
+            elif looped_len < 7:
+                if query_keys[looped_len] in tmp_str:
+                    looped_keys.append(query_keys[looped_len])
+                else:
+                    tmp_strlist = tmp_str.split(": ")
+                    config_meta_map[query_keys[looped_len - 1]][tmp_strlist[0]] = tmp_strlist[1]
+            elif looped_len == 7:
+                if (
+                    "ASM" in tmp_str
+                    or "CXX" in tmp_str
+                    or "DEBUG" in tmp_str
+                    or "HWLOC" in tmp_str
+                    or "LIBDL" in tmp_str
+                    or "LIBRT" in tmp_str
+                ):
+                    tmp_strlist = tmp_str.split(": ")
+                    config_meta_map[query_keys[looped_len - 1]][tmp_strlist[0]] = tmp_strlist[1]
+                elif query_keys[looped_len + 1] in tmp_str:
+                    if "Serial" in tmp_str:
+                        config_meta_map[query_keys[looped_len]] = "Serial"
+                        config_meta_map[query_keys[looped_len + 1]] = "Serial"
+                    else:
+                        tmp_strlist = tmp_str.split(" ")
+                        config_meta_map[query_keys[looped_len]] = tmp_strlist[0]
+                        config_meta_map[query_keys[looped_len + 1]] = info_str_list[i + 1]
+
+        return config_meta_map
+    """
+
```

Comment:
I agree with @chaeyeunpark regarding parsing the information in the C++ layer.
Thanks for having left both options.